src/spikeinterface/preprocessing/decimate.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+from spikeinterface.core.core_tools import (
+    define_function_from_class,
+)
+
+from .basepreprocessor import BasePreprocessor
+from .filter import fix_dtype
+from ..core import BaseRecordingSegment
+
+
+class DecimateRecording(BasePreprocessor):
+    """
+    Decimate the recording extractor traces using array slicing
+
+    Important: This uses simple array slicing for decimation rather than eg scipy.decimate.
+    This might introduce aliasing, or skip across signal of interest.
+    Consider  spikeinterface.preprocessing.ResampleRecording for safe resampling.
+
+    Parameters
+    ----------
+    recording : Recording
+        The recording extractor to be re-referenced
+    decimation_frame_step : int
+    decimation_frame_start : int, default 0
```

Comment:
```suggestion
    decimation_frame_step : int
    decimation_frame_start : int, default: 0
```
We use `default: ...`  for default values